### PR TITLE
Dance page - remove hyperlink highlight from the chevron icon

### PR DESCRIPTION
**Pull request by nkiruka**
To maintain style consistency throughout the website, the hyperlink highlight is removed from the chevron icon.

### Screen shots

- Before
image 3

- After
image

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/dance.haml [hunk 1]
```diff
@@ -10,6 +10,7 @@ theme: responsive
   %a{href: "/translate/dance"}
     %div.dance-translate-link-text
       =I18n.t(:hoc2018_dance_party_subheading)
+  %a{href: "/translate/dance", style: "text-decoration: none;"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/dance.haml]: Can you put the text and chevron in the same a tag? Then you can remove the underline from both and move in-line style to your css stylesheet.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: I initially had the code that way and intended to apply a text-decoration:none.  
@tanyaparker We want the underline just for the text right?  

@Erin007 This was the reason why I had to add the a tag to both the text and icon.

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/dance.haml]: We don't underline the text in other places where we use teal chevron links, such as /home, /projects and /congrats for example.  They should probably be consistent.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: https://studio.code.org/courses?view=teacher  (example of where the text is underlined).

I am ok with removing the underline completely.  @tanyaparker is this ok?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/dance.haml]: I guess we are already inconsistent in our styles. Let's remove the underline completely.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: sounds good!

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @Erin007 I refactored the code but I still have inline styling to remove the underline from the text and icon.  I extracted the style and applied it to the a tag but no change was observed.

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/dance.haml]: Did you apply your styling to `.dance-translate-link-text` ? You'll need to do it directly to the a tag but it should still be possible to do in your stylesheet.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @Erin007 I did, I try it again, if it doesn't work, I will meet you in person.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @Erin007 please review

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/dance.haml]: Can you update your screenshots, please? And address the merge conflict.

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance.css [hunk 2]
```diff
@@ -37,6 +37,10 @@
   clear: both;
 }
 
+<<<<<<< HEAD
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/css/dance-landing/dance.css]: You've committed some merge conflicts

File: pegasus/sites.v3/code.org/public/dance.haml (hunk 1 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @Erin007 Ready for review, thanks

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance.css (hunk 2 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/css/dance-landing/dance.css]: @tanyaparker Thanks good catch!

**Pull request merged**